build.msbuild

```diff
@@ -1,17 +1,38 @@
-<Project>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <ItemGroup>
         <VersionFile Include="version.props"/>
     </ItemGroup>
     <PropertyGroup>
         <NuspecPath>Microsoft.jQuery.Unobtrusive.Ajax.nuspec</NuspecPath>
+        <DistDir>$(MSBuildThisFileDirectory)dist\</DistDir>
+        <ArtifactsDir>$(MSBuildThisFileDirectory)artifacts\build\</ArtifactsDir>
+        <WebPackTestDir>$(MSBuildThisFileDirectory)test\webpacktest\</WebPackTestDir>
     </PropertyGroup>
+
+    <ItemGroup>
+        <DistDirFiles Include="$(DistDir)*.*;"/>
```

Comment:
I'm curious about the use of `$(DistDir)*.*;` instead of just `$(DistDir)*`. Is it possible this could be simplified, or is this glob format needed?